bin/ci-generate.py

```diff
@@ -12,31 +12,32 @@
 import datetime
 from jinja2 import Environment, FileSystemLoader
 import re
+import os
 
 
 # global variables
 REPO_PATH = ''
 VERBOSE = False
 OUTPUT_PATH = ''
 
-def load_objects(file_path, VERBOSE):
-    files = []
-    test_files = path.join(path.expanduser(REPO_PATH), file_path)
-    for file in sorted(glob.glob(test_files)):
-        if VERBOSE:
-            print("processing test: {0}".format(file))
-        files.append(load_file(file))
-    return files
-
-
-def load_file(file_path):
-    with open(file_path, 'r') as stream:
-        try:
-            file = list(yaml.safe_load_all(stream))[0]
-        except yaml.YAMLError as exc:
-            print(exc)
-            sys.exit("ERROR: reading {0}".format(file_path))
-    return file
+# def load_objects(file_path, VERBOSE):
```

Comment:
Are these load functions not needed any more?  Could we port them to a utility file for reuses in other parts of the app?